apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.Snippets/UrlSignerSnippets.cs

```diff
@@ -64,6 +64,41 @@ public async Task SignedURLGet()
             Assert.Equal(_fixture.HelloWorldContent, content);
         }
 
+        [Fact]
+        public async Task HmacSignedURLGet()
+        {
+            var bucketName = _fixture.BucketName;
+            var objectName = _fixture.HelloStorageObjectName;
+            var credential = (await GoogleCredential.GetApplicationDefaultAsync()).UnderlyingCredential as ServiceAccountCredential;
+            var httpClient = new HttpClient();
+
+            using var storageClient = await StorageClient.CreateAsync();
```

Comment:
I wonder whether this should be in the sample?

Reply:
We have plenty of samples for managing HMAC keys and a docs page explaining (I've linked to it on the text that accompanies this example). So:

- I'd prefer not to duplicate this as a sample.
- Don't give the appearance of the need to create an HMAC key each time. They may be stored in files, secret mamangers, etc.

Let me know.

Reply:
As a side note, I'll add deletion of HMAC keys to the CleanData project. Only 5 can be created, and if deletion fails, then we'll reach quota soon.

And delete on finally here as well.

Reply:
All fine by me.